### PR TITLE
contribution budget doc

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/286566495

Changes proposed in this pull request:

- Create ARA contribution budget doc
- https://pr-6567-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/contribution-budget/
- source: https://docs.google.com/document/d/1bU0a_njpDcRd9vDR0AJjwJjrf3Or8vAzyfuK8JZDEfo/edit#bookmark=id.ogwovwz4ufo7

### CODE REVIEW

File: site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- For a single conversion event (trigger), the adtech company can register multiple aggregatable values, and their sum should be at most 65,536 (`CONTRIBUTION_BUDGET`). If the sum of all aggregatable values for that single conversion exceeds this budget, the aggregatable report is not created.
+
+## Tips for success
+
+These tips will help you work successfully with the contribution budget.
+
+- Ensure your values are strictly within the contribution budget to not lose any information, since the budget is a hard cap. 
+- Allocate the contribution budget across metrics. Because the budget is common to all metrics for a given source, if more than one metric is tracked, you should share the budget across these metrics.
+
+- For example,if you're tracking two metrics—for example, the purchase value and the purchase count—you would split the contribution budget across these two metrics. More details in the example.
```
Comment (maudnals) [inline site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md:44]:
```suggestion
- If you're tracking two metrics—for example, the purchase value and the purchase count—you would split the contribution budget across these two metrics. More details in the example.
```

+ Missing link to example?

File: site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  Learn about the role of the contribution budget and how to allocate it to capture the data you need.  
+description: >
+  Learn about the role of the contribution budget and how to allocate it to capture the data you need.  
+date: 2022-03-01
+updated: 2023-03-14
+authors:
+  - maudn
+---
+
+To protect user privacy, individual users' contributions must have an upper limit.
```
Comment (maudnals) [inline site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md:14]: Could we add a one-liner that specifies the context i.e. summary reports in Attribution Reporting?

File: site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+In practice, this works as follows: across all aggregatable values associated with a single source (ad click or view), no value can be higher than a certain cap. We refer to this value as the `CONTRIBUTION_BUDGET`.
+
+## Contribution budget current value
+
+In the [explainer](https://github.com/WICG/conversion-measurement-api/blob/main/AGGREGATE.md), the contribution budget is referred to as the L1 budget. L1 budget = `CONTRIBUTION_BUDGET`. 
+
+For the current origin trial (started in Chrome 101), `CONTRIBUTION_BUDGET = 216 = 65,536`.
```
Comment (maudnals) [inline site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md:22]:
```suggestion
At the time of this writing, `CONTRIBUTION_BUDGET = 216 = 65,536`.
```
Outdated

**Review by maudnals — COMMENTED** [review]
Added comments

File: site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md:14]: done

**Review by anusmitaray — APPROVED** [review]
Left a comment with the space issue.

File: site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Contribution budget for summary reports'
+subhead: >
+  Learn about the role of the contribution budget for Attribution Reporting summary reports and how to allocate it to capture the data you need.  
+description: >
+  Learn about the role of the contribution budget for Attribution Reportingsummary reports and how to allocate it to capture the data you need.
```
[the remark this answers is not in the document]
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/contribution-budget/index.md]: done

**Pull request merged**